web/js/front.js

```diff
@@ -89,6 +89,7 @@ document.getElementById('pc').focus();
     var cta = document.getElementById('report-cta');
     if (cta && cta.addEventListener) {
         lk.setAttribute('role', 'button');
+        lk.setAttribute('tabindex', '0');
```

Comment:
Again, think should be the `if` above.
I think keyboard focusable is fine, makes it more consistent with the mouse-based behaviour, doesn't it.